src/mem/slab.h

```diff
@@ -33,7 +33,7 @@ namespace snmalloc
     alloc_new_list(void*& bumpptr, FreeListIter& fast_free_list, size_t rsize)
     {
       void* slab_end =
-        pointer_align_up<SLAB_SIZE>(pointer_offset(bumpptr, rsize));
+        pointer_align_up<SLAB_SIZE>(pointer_offset(bumpptr, 1));
```

Comment:
This isn't actually a bug (with the correction below in place), yes?  If the precondition for `alloc_new_list` is that `bumpptr` is at least `rsize` before the end of the slab (and I think that's the case?), then these will both produce the same `slab_end`, right?

Reply:
Yeah, this isn't needed.  It was part of my trying to chase down the but, but I thought it was perhaps nicer with 1, rather than `rsize`

Reply:
I'm happy to have it use `1` rather than `rsize`, I just wanted to check my understanding